lib/rack/geofilter.rb

```diff
@@ -15,26 +19,17 @@ def call env
 
   private
 
-  def filter_enabled?
-    ENV.has_key?('BLOCKED_COUNTRIES')
-  end
-
-  def countries_blocked
-    ENV['BLOCKED_COUNTRIES'].split(',')
-  end
-
-  def origin_country(env)
-    env['HTTP_CF_IPCOUNTRY']
-  end
-
-  def request_tagged?(env)
-    env.has_key?('HTTP_CF_IPCOUNTRY')
+  def setup_filter country_string
+    if !country_string.nil? && country_string != ""
```

Comment:
I was not 100% as to wether or not I'd have access to the `blank?` and `present?` methods in middleware.